source/server/admin/stats_html_render.cc

```diff
@@ -76,5 +76,18 @@ void StatsHtmlRender::noStats(Buffer::Instance& response, absl::string_view type
   }
 }
 
+// We override the generate method for HTML to trigger some JS that will
```

Comment:
Shouldn't this comment stay in the header? The header is usually where we go to find a description of what a function does.
Also it probably should be updated to reflect the updated behavior.

Reply:
Good point about updating it to the changed functionality :) But this is a virtual function override to just render the histogram, and (e.g.) text and prometheus and json all have different approaches to this.

Reply:
But this *specific* virtual function override is declared in `stats_html_render.h`, right? So there'll only be one implementation associated with that declaration. I'm not saying the comment belongs on the base class, but on the declaration for the function (where it was before).

Reply:
I don't feel that strongly about it, but I think as a reader I'd prefer to see it at the implementation; usually `override` headers do not have comments about how that implementation differs, unless the impl is inline. I decided not to inline this one since it's got a conditional in it and it feels more aesthetic to me in the .cc file.

I don't think the Envoy style guide is specific about this, is it?